Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -368,7 +372,8 @@ WarpX::computePhiCartesian (const amrex::Vector<std::unique_ptr<amrex::MultiFab>
                             int const verbosity) const
 {
 
-    // Define the boundary conditions
+#ifndef AMREX_USE_EB
+    // Without embedded boundaries: set potential at the box boundary
```

Comment:
Could this `if` block be moved down to line 406 (right before defining the `linop` object? That way the functionality to set domain boundary potentials can be preserved (it also requires moving the line `linop.setDomainBC( lobc, hibc );` out of the if block).